.travis.yml

```diff
@@ -1,11 +1,44 @@
+# WARNING: DO NOT EDIT!
+#
+# This file was generated by plugin_template, and is managed by bootstrap.py. Please use
+# bootstrap.py to update this file.
```

Comment:
Is this file maintained by bootstrap.py? If not, we should remove this header.

Reply:
it isn't, I just copied from pulp_file, but forgot to remove it